casper/src/test/scala/coop/rchain/casper/HashSetCasperTest.scala

```diff
@@ -530,7 +537,7 @@ class HashSetCasperTest extends FlatSpec with Matchers {
     nodes(1).casperEff.contains(signedBlock4) should be(true) // However, in invalidBlockTracker
 
     nodes(1).logEff.infos.count(_ startsWith "Added admissible equivocation") should be(1)
-    nodes(1).logEff.warns.count(_ startsWith "Recording invalid block") should be(1)
+    nodes(1).logEff.warns.count(_ startsWith "Recording invalid block") should be(0)
```

Comment:
Because we add the block created by itself directly, we will not mark the signedBlock4 invalid. And other nodes will still add the signedBlock4 except for those who contain the signedBlock1, who will find it is equivocation block. Is it right to do this?

Reply:
I would do like you have done above where you change the check to look at a different node which does do that validation.